src/server/frontend/render.js

```diff
@@ -22,7 +23,10 @@ export default function render(req, res, next) {
       isMobile: ['phone', 'tablet'].indexOf(req.device.type) > -1
     }
   }).toJS();
-  const store = configureStore({initialState: state});
+  const store = configureStore({
+    initialState: state,
```

Comment:
I get `Error: Invariant Violation: onlyChild must be passed a children with exactly one child.` with the actual version of este. Did you tried it with the actual version?